src/libopensc/card-isoApplet.c

```diff
@@ -67,6 +71,23 @@ static struct sc_card_driver isoApplet_drv =
 	NULL, 0, NULL
 };
 
+static struct isoapplet_supported_ec_curves {
+		struct sc_object_id oid;
+		size_t size;
+		unsigned int min_applet_version;
+} ec_curves[] = {
+	{{{1, 2, 840, 10045, 3, 1, 1, -1}},     192, 0x0000}, /* secp192r1, nistp192, prime192v1, ansiX9p192r1 */
```

Comment:
The only duplication is the order of the curve or key_length  in bits, (refered to as size in the structure) The array and version number are there to say which curve the applet supports, so this file will still need a table of OIDs with flags.